### DISCUSSION TITLE
Creating a step-by-step guide using buttons

### DISCUSSION THREAD

**Discussion by Javafox98** [microsoft/MixedRealityToolkit-Unity#9785] [Q&A] [answered]
So I'm trying to have it so I can create a step-by-step guide in Unity using MRTK (2.6) in a similar fashion to 365 guides.  Is there a way a user has to press a button to go forward/back while linking the input to another panel showing different text? 

Aka, IF user presses button THEN new text on panel.  

Thanks,
Javafox98

**Comment by polar-kev** [comment, marked as answer]
Hi @Javafox98, it sounds like you might want to explore the OnClick event for the HoloLens2 pressable button prefab and have it change the text in your text mesh pro game object.
https://docs.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/features/ux-building-blocks/button?view=mrtkunity-2021-05#collider-based-buttons-1
